pkg/runtime/cli.go

```diff
@@ -168,6 +170,16 @@ func FromFlags() (*DaprRuntime, error) {
 		readBufferSize = DefaultReadBufferSize
 	}
 
+	var gracefulShutdownDuration time.Duration
+	if *daprGracefulShutdownSeconds == -1 {
+		gracefulShutdownDuration = defaultGracefulShutdownDuration
+	} else {
+		gracefulShutdownDuration = time.Duration(*daprGracefulShutdownSeconds) * time.Second
```

Comment:
Tests miss this case.

Reply:
For now `cli_test.go` has no test for values from flag. And I've made it easier to test flag value parse in pr dapr/dapr#3951. I'd like to add these test after it was merged to avoid conflict and repetitive work.